### ISSUE TITLE
use in a wordpress plugin?

### ISSUE THREAD

**Issue by eigood** [iexg/IEX-API#11]
I don't know the correct location to ask this question, but recently google finance removed free api access(at the end of August).  I've modified the plugin that used that, to instead use iex.

Would it be a problem if I sent those changes back to the upstream developer of that plugin?  If you're curious, I can provide the name of the plugin.  I'm hoping you guys don't care, but it's your server/api.

**Comment by evancarter-iex** [comment]
Can you let us know the name of the plugin?

**Comment by eigood** [comment]
https://wordpress.org/support/plugin/stock-quote/

**Comment by evancarter-iex** [comment]
We don't have any problem with that.

**Comment by joshuablackburn-iex** [comment]
Just to follow up on this, I saw there was some confusion regarding our API legal terms.  We will get that language simplified.  There are no concerns with users of our API charging for their own products, or including our data into products that have a fee.  We only intended for users to not charge solely for our data.  We will get that simplified.  IEX is a transparent and credible source of data.
